.github/workflows/npm-cd.yml

```diff
@@ -306,18 +278,25 @@ jobs:
               run: |
                 apk update
                 apk add redis git
+                node -v
             
             - name: install Redis and Python for ubuntu
               if: ${{ contains(matrix.build.TARGET, 'linux-gnu') }}
               run: |
                 sudo apt-get update
-                sudo apt-get install redis-server python3
+                sudo apt-get install redis-server python3 
 
-            - name: install Redis and Python for macos
+            - name: install Redis, Python for macos
               if: ${{ contains(matrix.build.RUNNER, 'mac') }}
               run: |
                 brew install redis python3
 
+            - name: Checkout
+              if: ${{  matrix.build.TARGET != 'aarch64-unknown-linux-musl'}}
```

Comment:
This will cause a lot of problems when you add macos runner to the matrix. Just a notice.